examples/qwen3.8-flash-next/README.md

```
@@ -0,0 +1,38 @@
+3. Run the finetuning example:
+
+```bash
+axolotl train examples/qwen3.8-flash-next/qwen3.8-flash-next-qlora.yaml
+```
+
+## TIPS
+
+- This config trains on text-only conversation data; no `pixel_values` are supplied, so the vision tower is not exercised.
+- Uncomment the `linear_attn.*` entries in `lora_target_modules` to also adapt the Gated DeltaNet projections.
+- Read more on how to load your own dataset at [docs](https://docs.axolotl.ai/docs/dataset_loading.html).
+- The dataset format follows the OpenAI Messages format as seen [here](https://docs.axolotl.ai/docs/dataset-formats/conversation.html#chat_template).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)